tests/kafkatest/tests/core/mini_test.py

```diff
@@ -0,0 +1,18 @@
+from ducktape.tests.test import Test
+from ducktape.mark.resource import cluster
+
+from kafkatest.services.zookeeper import ZookeeperService
+from kafkatest.services.kafka import KafkaService
+
+
+class MiniTest(Test):
+    def __init__(self, test_context):
+        super(MiniTest, self).__init__(test_context=test_context)
+
+        self.zk = ZookeeperService(test_context, 1)
+        self.kafka = KafkaService(test_context, 1, self.zk)
+
+    @cluster(num_nodes=2)
+    def test(self):
+        self.zk.start()
```

Comment:
If it's just as well, could we use kraft instead, or alternatively provide two parameterizations?

Reply:
Yeah, sorry, this slipped my radar. It's a good idea, I'll add it - but how can we push this PR to older versions if it has kraft? Would it have to be a separate PR for older versions?

Reply:
Added kraft - I'm not super familiar with how kraft support is implemented in kafka tests, so please let me know if this is the right way to do it. Tests passed fwiw.